### PR TITLE
More flexible KElbow colors that use default palette by default

### LINKED ISSUE #1159: Change elbow plot fit time to use color palette instead of hardcoded "g"
https://github.com/DistrictDataLabs/yellowbrick/blob/2d4e23648b33ad42e2e95b559e35ca6de2db5bb6/yellowbrick/cluster/elbow.py#L406

Would be nice if the fit time line, as well as the elblow line - (black/dark-grey? not sure where/if this is hardcoded) could be changed to use the global color palette instead of hardcoded values.

I'd like to be able to use the matplotlib palette 'dark_background', which makes it impossible to see the black elbow line:

yellowbrick_elbow_black

Thanks!

### PR DESCRIPTION
**Pull request by bbengfort**
Sets the colors in KElbow to cycle colors (C0, C1) to use the default
palette rather than hardcoding color codes. Also adds properties so that
users can set the metric, timing, and vline colors manually (but not
passed in as parameters to init)

This PR fixes #1159

I have made the following changes:

1. Set constants for the `TIMING_COLOR`, `METRIC_COLOR`, both set to cyclic palette colors `"C1"` and `"C0"`
2. Created a `colors` dictionary on the visualizer and properties to set and access the colors for timing, metric, and vline
3. Updating the drawing code to use these colors
4. Created a test for changing colors
5. Updated test baseline images

### Sample Code and Plot

To use the dark background palette in matplotlib:

```python
# Import modeling 
from yellowbrick.cluster import KElbowVisualizer
from sklearn.datasets import make_blobs
from sklearn.cluster import KMeans

# Use dark background - must come after YB import
import matplotlib.pyplot as plt
plt.style.use('dark_background')

# Visualize K-Elbow
X, _ = make_blobs()
oz = KElbowVisualizer(KMeans())

oz.fit(X)
oz.show()
```

t

To set colors manually:

```
# Import modeling 
from yellowbrick.cluster import KElbowVisualizer
from sklearn.datasets import make_blobs
from sklearn.cluster import KMeans

# Visualize K-Elbow
X, _ = make_blobs()
oz = KElbowVisualizer(KMeans())

# Set colors
oz.metric_color = 'r'
oz.timing_color = 'c'
oz.vline_color = 'y'

oz.fit(X)
oz.show()
```

t

### TODOs and questions

No TODOs, all set here!

### CHECKLIST

- [x] _Is the commit message formatted correctly?_
- [ ] _Have you noted the new functionality/bugfix in the release notes of the next release?_

- [x] _Included a sample plot to visually illustrate your changes?_
- [x] _Do all of your functions and methods have docstrings?_
- [x] _Have you added/updated unit tests where appropriate?_
- [x] _Have you updated the baseline images if necessary?_
- [x] _Have you run the unit tests using `pytest`?_
- [x] _Is your code style correct (are you using PEP8, pyflakes)?_
- [ ] _Have you documented your new feature/functionality in the docs?_

- [ ] _Have you built the docs using `make html`?_

### CODE REVIEW

File: yellowbrick/cluster/elbow.py [hunk 1]
```diff
@@ -403,9 +421,33 @@ def finalize(self):
...
         if self.timings:
             self.axes[1].grid(False)
-            self.axes[1].set_ylabel("fit time (seconds)", color="g")
-            self.axes[1].tick_params("y", colors="g")
-
+            self.axes[1].set_ylabel("fit time (seconds)", color=self.timing_color)
+            self.axes[1].tick_params("y", colors=self.timing_color)
+
+
+    @property
```
Comment (lwgray) [inline yellowbrick/cluster/elbow.py:428]: Clever use of the properties here!

**Review by lwgray — APPROVED** [review]
This is ready to ship :shipit:

**Comment by bbengfort** [comment]
@lwgray no idea how I got the tests to pass ... but they did; so going to merge before I blink and they fail again!
